docs/security/hardening-octopus.md

```diff
@@ -125,7 +125,11 @@ Write-Output "Setting Windows Update to 'Download updates but let me choose whet
 Write-Output "This value allows Windows Defender to download and install definition updates automatically, but other updates are not automatically installed."
 cscript C:\Windows\System32\Scregedit.wsf /AU 3
 
-Write-Output "Excluding the Tools folder (e.g. Calamari) from Windows Defender..."
+Write-Output "Excluding the Tools folder for Octopus Server (e.g. Calamari) from Windows Defender..."
+Add-MpPreference -ExclusionPath "C:\Octopus\OctopusServer\Tools"
```

Comment:
Does this location depend on where a customer has chosen for the octopus server location for the specified instance?

Reply:
Yes, if the customer has chosen to put their Octopus Home folder into a different location other than the C drive it will be different but that whole malware script points to C inc the old tools folder for the tentacles.

In the documentation above that script it states:

`Note: you may need to change the excluded folders/files if you install Octopus Server or Tentacle into a different location.`

So we are covered here.